applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -581,16 +581,17 @@ static void gnss_event_handler(int event)
 		on_gnss_evt_pvt();
 		break;
 	case NRF_MODEM_GNSS_EVT_NMEA:
-		on_gnss_evt_nmea();
+		if (IS_ENABLED(CONFIG_SLM_LOG_LEVEL_DBG)) {
+			on_gnss_evt_nmea();
+		}
```

Comment:
This `if` should be removed, it has been replaced by the `#if defined` above.
```suggestion
		on_gnss_evt_nmea();
```